helix-term/src/ui/mod.rs

```diff
@@ -199,16 +199,22 @@ pub fn file_picker(root: PathBuf, config: &helix_view::editor::Config) -> Picker
     });
     log::debug!("file_picker init {:?}", Instant::now().duration_since(now));
 
-    let picker = Picker::new(Vec::new(), root, move |cx, path: &PathBuf, action| {
-        if let Err(e) = cx.editor.open(path, action) {
-            let err = if let Some(err) = e.source() {
-                format!("{}", err)
-            } else {
-                format!("unable to open \"{}\"", path.display())
-            };
-            cx.editor.set_error(err);
-        }
-    })
+    let picker_title = format!("Files in {:?}", root.as_os_str());
```

Comment:
This will also have the length problem and should probably be collapsed, only expand the last N directories, if it reaches the width limit of the windows